tests/Unit/Core/Addon/Theme/ThemeValidatorTest.php

```diff
@@ -91,6 +91,12 @@ private function getTheme(string $name = 'valid'): Theme
         $themeDir = __DIR__ . '/../../../../Resources/themes/minimal-' . $name . '-theme/';
         $themeConfigFile = $themeDir . 'config/theme.yml';
 
+        try {
+            $themeConfigContent = file_get_contents($themeConfigFile);
+        } catch (\Throwable $exception) {
+            throw new \RuntimeException(sprintf('Unable to read theme config file %s', $themeConfigFile));
+        }
```

Comment:
Hi @justeen35. Thanks for this PR.

I think it's not usefull since we are in a test. It doesn't matter which kind of error we throw.
Or maybe it's to have a more explicit message?

Reply:
Different people have different opinions about this.

I treat my code in `tests/` just like I treat my code in `src` : I apply it same rules, same checks, same syntax, same requirements. I run phpstan on my tests folder too. It's still code, it needs to evolve, it needs to be debugged, it needs to be developer friendly.

You are free to choose not to merge this pull request